### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by alymdrictels**
Branch to introduce tests and some refactoring/code relocation to support the tests

### CODE REVIEW

File: Tests/Helper/UserContentHelperTest.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
```
Comment (iherak) [inline Tests/Helper/UserContentHelperTest.php]: This file is part of the HWIOAuthBundle?

File: phpunit.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<phpunit bootstrap="vendor/autoload.php"
+         backupGlobals="false"
+         backupStaticAttributes="false"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         colors="true"
+>
+    <testsuites>
+        <testsuite name="Netgen\EzFormsBundle\Tests">
```
Comment (iherak) [inline phpunit.xml]: ezformsbundle?

Comment (alymdrictels) [reply phpunit.xml]: This is why I say XML is not human-readable.

File: Tests/Helper/UserContentHelperTest.php (hunk 1 above)
Comment (alymdrictels) [reply Tests/Helper/UserContentHelperTest.php]: Fixed.

**Pull request merged**